src/v0/destinations/klaviyo/util.js

```diff
@@ -30,7 +29,6 @@ const {
   MAX_BATCH_SIZE,
   WhiteListedTraitsV2,
   revision,
-  revisionOct2024,
 } = require('./config');
 const logger = require('../../../logger');
 
```

Comment:
Can REVISION_CONSTANT variable be cleaned up ? what's this version about ?

Reply:
`REVISION_CONSTANT` is being used the older (2023-02-22) API implementation.